qa/tasks/cephfs/test_fscrypt.py

```diff
@@ -83,9 +83,10 @@ def verify_alternate_name():
         self.fs.set_joinable()
         self.fs.wait_for_daemons()
 
+        self.mount_a.run_shell_payload(f"cd {self.path} && find")
```

Comment:
IMO, it'll be helpful to place a comment here explaining why we need this before calling `verify_alternate_name()`. Perhaps something like `# Add dirfrag to memory/cache since rank will not have it because journal was reset`. The commit message is pretty good, so something like it.

I am not sure if it's worth it but, should we consider moving this line to inside `verify_alternate_name()`? Placing this line next to first line in there (`J = self.fs.read_cache("/dir", depth=0`) makes the context of this line clearer.

Reply:
I've added a comment. I don't think the belongs in `verify_alternate_name` which is just doing introspective commands on the MDS.